### PR TITLE
Fixed member validation

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:
We need to ensure that there is at least an empty validation defined as the fields are visible but disabled in case of `update` dialog.
We also cannot just remove the return as we must not use the `add`  validations for `update` dialog. In this case, the dialog would always return errors upon save like 'user foo already exists'

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```bugfix user
Fixed an issue on the member management page. Update members and service account dialog did not render correctly because of an issue with the input validation
```

### CODE REVIEW

File: frontend/src/components/Members/GMemberDialog.vue [hunk 1]
```diff
@@ -240,6 +240,7 @@ export default {
     rules.internalRoles = withFieldName('Member Roles', internalRolesRules)
 
     if (this.isUpdateDialog) {
+      rules.internalName = {}
```
Comment (petersutter) [inline frontend/src/components/Members/GMemberDialog.vue:243]: instead, initialize it already here

```js
const rules = {
  internalName: {}
}
```

Comment (holgerkoser) [reply frontend/src/components/Members/GMemberDialog.vue:243]: I still do not understand why not just remove the `if (this.isUpdateDialog) {`

Comment (grolu) [reply frontend/src/components/Members/GMemberDialog.vue]: Added comment to make it clear

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Review by petersutter — APPROVED** [review]
/lgtm

**Pull request merged**